compose/install/index.md

```diff
@@ -3,30 +3,73 @@ description: How to install Docker Compose
 keywords: compose, orchestration, install, installation, docker, documentation
 title: Install Docker Compose
 toc_max: 2
+redirect_from:
+- /compose/compose-desktop/
 ---
 
-On this page you can find a summary of the available options for installing Docker Compose. 
+On this page you can find information on how to get and install Compose.
```

Comment:
```suggestion
```

I think this page is short and self-evident enough not to require this explanation.

Reply:
Agree. Just keeping what we're doing as a standard in the docs. Usually it helps with the weirdness of having content start right after the page's title (from a visual perspective).

Reply:
See closing comment.